pkg/controllers/leaderworkerset_controller.go

```diff
@@ -139,13 +138,20 @@ func (r *LeaderWorkerSetReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	}
 
 	if !lwsUpdated {
-		// in the cases where lws is updated, the Partition in leaderSts is zero, not a useful event to record
+		// An event is logged to track update progress.
 		if leaderSts != nil && partition != *leaderSts.Spec.UpdateStrategy.RollingUpdate.Partition {
-			r.Record.Eventf(lws, corev1.EventTypeNormal, "GroupsAreUpgrading", fmt.Sprintf("Upgrading replicas %d to %d", *leaderSts.Spec.UpdateStrategy.RollingUpdate.Partition, partition))
+			r.Record.Eventf(lws, corev1.EventTypeNormal, GroupsAreUpdating, fmt.Sprintf("Updating replicas %d to %d", *leaderSts.Spec.UpdateStrategy.RollingUpdate.Partition, partition))
 		}
 	}
 
+	if leaderSts == nil {
```

Comment:
sorry, this should be after the call to `SSAWithStatefulset` if we are to be consistent with the event we log for the workers sts

Reply:
and s/Creating/Created